modules-js/react-fleet/src/form-elements/inputs/MemorableDateInput.test.ts

```diff
@@ -0,0 +1,188 @@
+import { isDateValid, setDateLimit } from './MemorableDateInput';
+
+const today = new Date(new Date().setHours(0, 0, 0, 0));
```

Comment:
Lock this in to being a specific date, otherwise you put a timebomb in the code when tests start failing next year.